include/shad/util/measure.h

```diff
@@ -56,13 +56,14 @@ struct measure {
   static auto duration(F&& function, Args... args) {
     auto start = std::chrono::steady_clock::now();
     std::forward<decltype(function)>(function)(std::forward<Args>(args)...);
-    auto duration = std::chrono::duration_cast<TimeT>(
-        std::chrono::steady_clock::now() - start);
+    std::chrono::duration<double, typename TimeT::period> duration =
+                   std::chrono::steady_clock::now() - start;
 
     return duration;
   }
 };
 
+
```

Comment:
Can you remove this extra space?  Did you run make clang-format?

Reply:
done